features/steps/consistency.py

```diff
@@ -75,9 +75,9 @@ def step_impl(context, image, output_format):
     output_split = output_switch.split(' ')
     # there's probably a more pythonic way to do this, with array flattening etc.
     # just being expedient for the time being
-    args = [found_executable, '-tiff', output_split[0], image]
+    args = [found_executable, '-tiff', '-color', 'none', '-no-denoise', output_split[0], image]
     if len(output_split) == 2:
-        args = [found_executable, '-tiff', output_split[0], output_split[1], image]
+        args = [found_executable, '-tiff', '-color', 'none', '-no-denoise', output_split[0], output_split[1], image]
```

Comment:
OK - now it is 2 o'clock in the middle of the night and I got it working.

I am going to dive into my bed .

Nice reading :)